pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -661,13 +661,14 @@ func TestListPodResourcesWithInitContainersV1(t *testing.T) {
 						Name:      podName,
 						Namespace: podNamespace,
 						Containers: []*podresourcesapi.ContainerResources{
-							{
-								Name:             initContainerName,
-								Devices:          devs,
-								CpuIds:           cpus,
-								Memory:           memory,
-								DynamicResources: []*podresourcesapi.DynamicResource{},
-							},
+							// @haircommander: this causes the test to fail, not sure why
```

Comment:
@ffromani do you expect this? I couldn't reason easily what was going on

Reply:
theory: this code predates the DRA/podresources bridging code. Will have a closer look and get back.

Reply:
turns out the likely case is actually u/s dd890b899fa . Removing the latest commit (which commentss the affected test) and adding dd890b899fa to backport make the test pass cleanly.

I think the correct approach in this case is to document the reason why we want to skip this test, and keep skipping it. Otherwise we need to pull in sidecar container related changes, which seems undesirable.